build.gradle

```diff
@@ -194,6 +194,27 @@ task run(type: JavaExec, dependsOn: 'build') {
         '-Dbc.server.save-file=' +project.property('replay')
     ]
 }
+task playtest(type: JavaExec, dependsOn: 'build') {
```

Comment:
This adds the playtest command to gradle. I made a copy of the "run" command and turn debugging off. To minimize the console output for the playtest report. The playtest report os a whole other conversation. With all the System.out.println that is in our bot, the playtest report is 80mb compressed.